modules/pinger/src/test/java/org/hawkular/component/pinger/test/PingerTest.java

```diff
@@ -34,27 +35,51 @@
  */
 public class PingerTest {
 
-    @org.junit.Test
+    @Test
     public void testPinger() throws Exception {
 
         Pinger pinger = new Pinger();
         PingDestination destination = new PingDestination("123","http://hawkular.github.io");
         PingStatus status = pinger.ping(new PingStatus(destination)).get();
 
-        assert status.getCode()==200;
-        assert status.isTimedOut()==false;
+        Assert.assertEquals(200, status.getCode());
+        Assert.assertFalse(status.isTimedOut());
+
+    }
+
+    @Test
+    public void testHeadPinger() throws Exception {
+
+        Pinger pinger = new Pinger();
+        PingDestination destination = new PingDestination("123","http://hawkular.github.io", "HEAD");
```

Comment:
I do not see such a need in this test. 
In `PingDestination`, there is a constant for the default method `"GET"` and there is no other method named in non-test code.